docs/sections.md

```diff
@@ -20,7 +20,7 @@ Sections help to maintain vertical rhythm.
 </div>
 
 ```html
-<div class="section">
+<div class="section section--first">
```

Comment:
Using a single class that has a fixed value for spacing will help to enforce consistency.